FluentAssertions.Core/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -101,7 +101,7 @@ public class EquivalencyAssertionOptions : EquivalencyAssertionOptionsBase<Equiv
         public EquivalencyAssertionOptions()
         {
             IncludingNestedObjects();
-            IncludingAllDeclaredProperties();
+            IncludingAllDeclaredMembers();
```

Comment:
I had expected this was going to need some discussion.  I am glad to hear this can make it in >400,000 downloads on NuGet is nothing to scoff at; I would hate to bring about changes or instability that would turn people away.  But at the same am trying to make it the best I can going forward too.

While I was thinking this would not make it I thought through a 'next-best-option' and an it is still stuck in my mind that it might still be a good idea in addition to this.  I will start a discussion on that elsewhere.